docs/source/merlin_specification.rst

```diff
@@ -277,6 +277,14 @@ see :doc:`./merlin_variables`.
               batch:
                  type: local
               machines: [host3]
+  ####################################
+  # Merlin Block (Required)
+  ####################################
+  # The user block allows other variables in the workflow file to be propagated
```

Comment:
Maybe make a note that this is optional, because it looks like it is required from the comment above.

Reply:
I have updated the documentation on the user block and added the example from the freature_demo to the docs.